density/db/db.py

```diff
@@ -189,7 +189,7 @@ def get_oauth_code_for_uni(cursor, uni):
     else:
         # If the code DNE, create a new one and insert into the database.
         new_code = ''.join(random.choice(
-            string.ascii_uppercase + string.digits) for x in xrange(32))
+            string.ascii_uppercase + string.digits) for _x in xrange(32))
```

Comment:
I personally prefer `__` (or `_`, but that sometimes conflicts with `_` as the last output in the repl) instead of `x`.

...We should also change this code. If it's for identification, we should use https://docs.python.org/2.7/library/uuid.html (either encoded as hex or as base64). If it's for security / crypto, we should rethink our lives and cry.

Related: do you mind adding a UNIQUE index + constraint onto the `oauth_data` schema? We might have to talk to @thebrianzeng about letting you run the build index command on the running postgres instance.